spec/requests/cheque_payment_forms_spec.rb

```diff
@@ -4,8 +4,9 @@
 
 RSpec.describe "ChequePaymentForms", type: :request do
   let(:transient_registration) do
-    create(:renewing_registration, :has_finance_details, :does_not_require_conviction_check)
```

Comment:
A standard `renewing_registration` object is currently created with existing payments. By using the trait, we were overriding the default behaviour by creating empty finance details. In order to reproduce the issue reported in the ticket, given https://github.com/rails/rails/issues/24796, I have removed the specific call to the trait so that we run our code on a collection that also contains other payments.